test/e2e/snaps/test-snap-uilinks.spec.js

```diff
@@ -0,0 +1,129 @@
+const {
+  defaultGanacheOptions,
+  withFixtures,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { TEST_SNAPS_WEBSITE_URL } = require('./enums');
+
+describe('Test Snap Dialog', function () {
+  it('test all three snap_dialog types', async function () {
```

Comment:
If this needs to be a separate test we should update all the of this

Reply:
I have made changes in the latest commit to reflect what is actually happening.